.github/workflows/build_linux_jax_wheels.yml

```diff
@@ -0,0 +1,117 @@
+name: Build Linux PyTorch Wheels
+
+on:
+  workflow_call:
+    inputs:
+      amdgpu_family:
+        required: true
+        type: string
+      python_versions:
+        required: true
+        type: string
+      release_type:
+        description: The type of release to build ("nightly", or "dev")
+        required: true
+        type: string
+      s3_subdir:
+        description: S3 subdirectory, not including the GPU-family
+        required: true
+        type: string
+      cloudfront_url:
+        description: CloudFront URL pointing to Python index
+        required: true
+        type: string
+      rocm_version:
+        description: ROCm version to install
+        type: string
+      tar_url:
+        description: URL to TheRock tarball to build against
+        type: string
+  workflow_dispatch:
+    inputs:
+      amdgpu_family:
+        required: true
+        type: string
+      python_versions:
+        required: true
+        type: string
+        default:
+      release_type:
+        description: The type of release to build ("nightly", or "dev")
+        required: true
+        type: string
+      s3_subdir:
+        description: S3 subdirectory, not including the GPU-family
+        type: string
+        default: "v2"
+      cloudfront_url:
+        description: CloudFront base URL pointing to Python index
+        type: string
+        default: "d25kgig7rdsyks.cloudfront.net"
+      rocm_version:
+        description: ROCm version to install (e.g. "==1.0", ">1.0")
+        type: string
+      tar_url:
+        description: URL to TheRock tarball to build against
+        type: string
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  build_jax_wheels:
+    strategy:
+      matrix:
+        jax_ref: [master]
+    name: Build Linux JAX Wheels | ${{ inputs.amdgpu_family }} | Python ${{ inputs.python_version }}
+    runs-on: ${{ github.repository_owner == 'ROCm' && 'azure-linux-scale-rocm' || 'ubuntu-24.04' }}
+    env:
+      PACKAGE_DIST_DIR: ${{ github.workspace }}/jax_rocm_plugin/wheelhouse
+      S3_BUCKET_PY: "therock-${{ inputs.release_type }}-python"
+    steps:
+      - name: Checkout JAX
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          repository: rocm/rocm-jax
+          ref: ${{ inputs.jax_ref }}
+
+      - name: Configure Git Identity
+        run: |
+          git config --global user.name "therockbot"
+          git config --global user.email "therockbot@amd.com"
+
+      - name: Build JAX Wheels
+        run: |
+          cd rocm-jax
+          python3 build/ci_build \
+            --compiler=clang \
+            --python-versions="${{ inputs.python_versions }}" \
+            --rocm-version="${{ inputs.rocm_version }}" \
+            --therock-tar-url="${{ inputs.tar_url }}" \
```

Comment:
Is the tar URL a baseurl or is it the absolute URL to a tarball? In that case, what does the rocm version argument do?

Reply:
This is not a problem for TheRock's CI, but maybe it is worth a add a comment here. Just to clarify things, `--rocm-vesion` however is needed in addition? If so, that's fine, otherwise I would have suggested to drop those inputs form the workflow.

Reply:
Sorry if I wasn't clear. Yes, `--rocm-version` is required in addition to `--therock-path`.